content/en/docs/challenge-types.md

```diff
@@ -150,6 +150,7 @@ Cons:
 
  - It’s not supported by Apache, Nginx, or Certbot, and probably won’t be soon.
  - Like HTTP-01, if you have multiple servers they need to all answer with the same content.
+ - Let’s Encrypt doesn’t let you use this challenge to issue wildcard certificates.
```

Comment:
Better to phrase this as something like "This method cannot be used to validate wildcard domains"

Reply:
The wording is taken verbatim from the same document further above. Want me to change both uses of it?

Reply:
Oh! Heh, I hadn't noticed that, thanks for pointing it out. Yes, changing both would be better -- this is forbidden by the Baseline Requirements section 3.2.2.4.20, not by Let's Encrypt.

Reply:
Alright, I've changed it appropriately.